src/modules/core/components/UserPickerWithSearch/UserPickerWithSearch.tsx

```diff
@@ -161,6 +162,11 @@ const UserPickerWithSearch = ({
       ? placeholder
       : formatMessage(placeholder);
 
+  const dropdownHeight = useMemo(() => {
+    const height = data.length * 56;
+    return height > 180 ? 180 : height;
```

Comment:
Can you please declare `56` and `180` as variables (you can do it outside of the component) and use variables? Not great to have just numbers.
Please, use capital letters for the variables, e.g. ELEMENT_HEIGHT